### PR TITLE
Create changelog for API release 4.65.0

### PR DESCRIPTION
**Pull request by bbiggerr**

### CODE REVIEW

**Comment by leslitagordita** [comment]
@bbiggerr I added my entries to the Changelog. Please review :)

File: src/content/changelog/4-65-0-2020-05-18.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+---
+
+### Added
+
+- The `managed_issue` field was added to the Open Support Ticket endpoint ([POST /support/tickets](/api/v4/support-tickets/#post)).
+    - This field designates if the Support ticket is in reference to a [Managed Service](https://www.linode.com/products/managed/).
+
+### Fixed
+
+- A bug was fixed that was preventing initiation of scheduled, cross-data center migrations.
```
Comment (alsaihn) [inline src/content/changelog/4-65-0-2020-05-18.md]:
```suggestion
- A bug was fixed that was preventing initiation of scheduled, cross-datacenter migrations.
```

File: src/content/changelog/4-65-0-2020-05-18.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+version: 4.65.0
+changelog:
+  - API
+---
+
+### Added
+
+- The `managed_issue` field was added to the Open Support Ticket endpoint ([POST /support/tickets](/api/v4/support-tickets/#post)).
+    - This field designates if the Support ticket is in reference to a [Managed Service](https://www.linode.com/products/managed/).
+
```
Comment (alsaihn) [inline src/content/changelog/4-65-0-2020-05-18.md:15]: Can we add a note about how you can't have the other ids present if this field is passed in?

**Review by alsaihn — CHANGES_REQUESTED** [review]
Two small things.

Also, we had a fix for our unsuspending workflow, but the LKE team understands that better than I do.

File: src/content/changelog/4-65-0-2020-05-18.md (hunk 1 above)
Comment (bbiggerr) [reply src/content/changelog/4-65-0-2020-05-18.md]: I'm seeing inconsistent usage between "data center" and "datacenter" in past changelog updates. Most international standards recommend "data center, and in the past we referred to this feature as "cross data center," so I recommend going with that.

Comment (alsaihn) [reply src/content/changelog/4-65-0-2020-05-18.md]: Ah, making it consistent would be nice, but I'm good with "cross data center" also. The cross-data part looked weird to me before.

**Review by alsaihn — APPROVED** [review]

**Pull request merged**